ansible/group_vars/all/system.yaml

```diff
@@ -13,13 +13,22 @@ docker_rpm_repository_url_upstream_mirror: "https://packages.d2iq.com/download.d
 docker_rpm_repository_url: "https://packages.d2iq.com/konvoy/stable/linux/centos/{{ ansible_distribution_major_version|int }}/x86_64"
 docker_rpm_gpg_key_url: "{{ docker_rpm_repository_url }}/gpg.asc"
 
-# nvidia package - when adding new urls, please add them to pre-flight checks
+# nvidia package
 libnvidia_container_rpm_repository_package_url: "https://nvidia.github.io/libnvidia-container/centos7/x86_64"
 libnvidia_container_deb_repository_url: "https://nvidia.github.io/libnvidia-container/ubuntu16.04/amd64"
 libnvidia_container_gpg_key_url: "https://nvidia.github.io/libnvidia-container/gpgkey"
 nvidia_container_runtime_rpm_repository_package_url: "https://nvidia.github.io/nvidia-container-runtime/centos7/x86_64"
 nvidia_container_runtime_deb_package_url: "https://nvidia.github.io/nvidia-container-runtime/ubuntu16.04/amd64"
 nvidia_container_runtime_gpg_key_url: "https://nvidia.github.io/nvidia-container-runtime/gpgkey"
+nvidia_driver_rpm_repository_package_url: http://developer.download.nvidia.com/compute/cuda/repos/rhel{{ ansible_distribution_major_version|int }}/x86_64/
+nvidia_driver_rpm_gpg_key_url: http://developer.download.nvidia.com/compute/cuda/repos/rhel{{ ansible_distribution_major_version|int }}/x86_64/7fa2af80.pub
+# On Centos7 installs `nvidia-driver` with dkms for kernel line 3.x w version
+nvidia_driver_package_name: "nvidia-driver-latest-dkms-3"
+nvidia_driver_package_version: "460.73.01-1.el7.x86_64"
+# On Centos8 installs `nvidia-driver` module with given version
+nvidia_driver_module_name: "nvidia-driver"
+nvidia_driver_module_version: "460-dkms"
+nvidia_driver_deb_repository_url: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/
```

Comment:
It's not immediately clear what the difference here is between `package` and `module`. Can we perhaps expand the names to `...rpm_package...` and `...dnf_module...`? That then matches the `...deb_repository...` pattern.

Reply:
Fixed.